### PR TITLE
[TF] Hide unnecessary model object from output. Fixes #1125

### LINKED ISSUE #1125: Unnecessary output <tensorflow.python.keras.engine.sequential.Sequential at 0x7fd94cc7aa50>
http://preview.d2l.ai.s3-website-us-west-2.amazonaws.com/d2l-en/master/chapter_convolutional-neural-networks/lenet.html

Is there a way to avoid printing unnecessary output for train_ch6 (and possibly other similar functions):
<tensorflow.python.keras.engine.sequential.Sequential at 0x7fd94cc7aa50>

@terrytangyuan

### PR DESCRIPTION
**Pull request by terrytangyuan**
*Description of changes:* Fixes #1125

By submitting this pull request, I confirm that you can use, modify,
copy, and redistribute this contribution, under the terms of your
choice.

### CODE REVIEW

File: chapter_convolutional-neural-networks/lenet.md [hunk 1]
```diff
@@ -403,11 +403,17 @@ def train_ch6(net_fn, train_iter, test_iter, num_epochs, lr,
...
-#@tab all
+#@tab mxnet,pytorch
 lr, num_epochs = 0.9, 10
 train_ch6(net, train_iter, test_iter, num_epochs, lr)
 ```
 
+```{.python .input}
+#@tab tensorflow
+lr, num_epochs = 0.9, 10
+model = train_ch6(net, train_iter, test_iter, num_epochs, lr)
```
Comment (astonzhang) [inline chapter_convolutional-neural-networks/lenet.md:414]: if `model` is not used, rename it to `_`

**Comment by mli** [comment]
Job d2l-en/PR-1221/1 is complete. 
Check the results at http://preview.d2l.ai/d2l-en/PR-1221/

**Comment by astonzhang** [comment]
Is it possible to edit implementations of `train_ch6` instead of assigning `train_ch6` to a variable? Then we can still reuse the same code across frameworks.

**Comment by terrytangyuan** [comment]
> Is it possible to edit implementations of `train_ch6` instead of assigning `train_ch6` to a variable? Then we can still reuse the same code across frameworks.

It's a bit complicated right now since it involves using `tf.keras` and GPUs.

**Comment by astonzhang** [comment]
In that case, let's keep what we have instead of assigning it to a variable. Closed https://github.com/d2l-ai/d2l-en/issues/1125

**Comment by mli** [comment]
Job d2l-en/PR-1221/2 is complete. 
Check the results at http://preview.d2l.ai/d2l-en/PR-1221/

**Comment by terrytangyuan** [comment]
Okay. Closing this now.

**Pull request closed without merging**